apps/api/tests/api/application-api.test.ts

```diff
@@ -232,6 +261,76 @@ describe('Application API', () => {
 		});
 	});
 
+	describe('Request Application Revisions', () => {
+		it('should request revisions when application is in DAC_REVIEW state', async () => {
+			const applicationRecordsResult = await testApplicationRepo.listApplications({ user_id });
+			assert.ok(applicationRecordsResult.success);
+			assert.ok(
+				Array.isArray(applicationRecordsResult.data.applications) && applicationRecordsResult.data.applications[0],
+			);
+			const { id } = applicationRecordsResult.data.applications[0];
+			const role = 'DAC';
+			const repId = '12345';
+			const comments = 'Please provide additional documentation.';
+
+			// Act: Call the function
+			const result = await requestApplicationRevisions({
+				applicationId: id,
+				role,
+				repId,
+				reviewData: JSON.stringify(revisionRequestData),
```

Comment:
These `reviewData: JSON.stringify(revisionRequestData)` are giving me Type errors and failing on `pnpm build:all`